### PR TITLE
chore: use preact as dom renderer

### PR DESCRIPTION
**Pull request by nilzona**
**Checklist**

- [x] tests added
- [x] commits conform to the [commit guidelines](./CONTRIBUTING.md#commit)
- [ ] documentation updated

### CODE REVIEW

File: packages/picasso.js/package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
   ],
   "main": "dist/picasso.js",
   "module": "src/index.js",
+  "source": true,
```
Comment (nilzona) [inline packages/picasso.js/package.json]: This is useful if you use Parcel to build a project that you `npm link` to picasso.js ... zero config ^^

File: packages/picasso.js/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+package-lock.json
```
Comment (stoffeastrom) [inline packages/picasso.js/.gitignore]: Why are we ignoring this?

File: plugins/q/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+package-lock.json
```
Comment (stoffeastrom) [inline plugins/q/.gitignore]: Why are we ignoring this?

File: packages/picasso.js/src/web/components/tooltip/render.js [hunk 4]
```diff
@@ -25,38 +26,21 @@ export default function render(data, placement, {
...
-      style: extend(tooltipDefaultStyle, placement.computedTooltipStyle),
-      class: extend({ 'pic-tooltip': true }, classes.tooltip),
-      attrs: {
-        dir: props.direction
-      }
-    },
-    [contentNode, arrowNode]
+  const tooltipNode = (
+    <div style={extend(tooltipDefaultStyle, placement.computedTooltipStyle)} class={classString(extend({ 'pic-tooltip': true }, classes.tooltip))} dir={props.direction}>
+      <div style={extend({}, style.arrow, style[`arrow-${placement.dock}`], placement.computedArrowStyle)} class={classString(extend({ 'pic-tooltip-arrow': true }, classes.arrow))}></div>
```
Comment (cbt1) [inline packages/picasso.js/src/web/components/tooltip/render.js]: To keep the order, the arrow div should be last

**Pull request merged**